### DISCUSSION TITLE
Scrypted Cloud and Cloudflare tunnel

### DISCUSSION THREAD

**Discussion by gt9228a** [koush/scrypted#1555] [Q&A]
I am trying to use the cloudflare tunnel option in the scrypted cloud plugin.

The instructions say: Paste the token into the Cloud Plugin Advanced Settings.

Where exactly is this pasted to?  Is it pasted under "Cloduflare tunnel custom domain"?
I see no where else to paste the token that I created from cloudflare tunnel.

Thank you for this great product!

**Comment by thatjuan** [comment]
I'm having the same issue. I believe the documentation is out-dated. I suspect we now have to set up the tunnel manually with a custom domain and then paste the custom domain in that field. I haven't tried this yet, though.
